src/session/sessionReducer.js

```diff
@@ -37,6 +37,11 @@ export type SessionState = {|
   isActive: boolean,
   isHydrated: boolean,
   lastNarrow: ?Narrow,
+
+  /**
+   * Whether the /register request is in progress; i.e., on startup, or on
+   * re-init following a dead event queue after 10 minutes of inactivity.
+   */
   loading: boolean,
```

Comment:
branch-level nit: put this commit before the one that adds `LoadingBanner`, because this helps explain what the latter is doing